src/types.ts

```diff
@@ -12,13 +12,32 @@ export type Post = {
   linkToSpecialtyPage?: string
 }
 
+export type CommunityEventCategory = 'tournament' | 'social' | 'learning' | 'generic'
+
+export type CommunityEventCollectionData = {
+  title: string
+  url: string
+  category: CommunityEventCategory
+  startDateTime: string
+  ignoreStartTime?: boolean
+  endDateTime?: string
+  ignoreEndTime?: boolean
+  host: string
+}
+
+export type CommunityEventHostCollectionData = {
+  name: string
+  url: string
+}
+
+export type CommunityEventHost = CommunityEventHostCollectionData
+
 export type CommunityEvent = {
```

Comment:
I'm not sure I like the CommunityEvent data type. We've adapted to it since it's been there, but we might want to change it. It's odd that we convert from a Date object to split string objects back to Date objects by joining them back together.

It has grown in this way, but it seems pretty silly in retrospect.

Reply:
I am unopposed to changing this but perhaps this could be done in a later clean up? I am also unsure of how to implement the change.

Reply:
Yeah, I don't see an issue with that. Makes merging with Jonas a bit easier as well for now.